cpp/server/ct-server.cc

```diff
@@ -202,11 +202,15 @@ void SequenceEntries(TreeSigner<LoggedCertificate>* tree_signer,
 
   while (true) {
     if (election->IsMaster()) {
-      store->CleanupOldEntries();
-
-      // TODO(alcutter): Probably don't need to blow up here
-      CHECK_EQ(util::Status::OK, tree_signer->SequenceNewEntries())
-          << "Problem sequencing new entries";
+      util::Status status(store->CleanupOldEntries());
+      if (!status.ok()) {
+        LOG(WARNING) << "Problem cleaning up old entries: " << status;
+      } else {
+        status = tree_signer->SequenceNewEntries();
```

Comment:
Only if there was no problem cleaning up? I agree these should be done one after the other (because they'll both try to modify the mapping entry), but it should be fine to at least attempt to sequence even if there was a problem cleaning up, I'd say.